ui/lib/utils/i18n.ts

```diff
@@ -37,18 +35,32 @@ export function addTranslationResource(lang, translations) {
 }
 
 export const ALL_LANGUAGES = {
-  zh: '简体中文',
+  'zh-CN': '简体中文',
   en: 'English',
 }
 
+export function getEffetiveLang(): string {
+  const effetiveLangs = Object.keys(ALL_LANGUAGES)
+  const detectedLang = i18next.language
+  if (effetiveLangs.includes(detectedLang)) {
+    return detectedLang
+  }
+  if (detectedLang.startsWith('zh')) {
+    return 'zh-CN'
+  }
+  return 'en'
+}
+
```

Comment:
Is this helpful? https://github.com/i18next/i18next/issues/489 I found others may meet similar problems.

Reply:
😂 Sorry, I didn't get the relationship between this method and issue i18next/i18next#489 .